pac4j-saml/src/main/java/org/pac4j/saml/logout/impl/SAML2LogoutRequestBuilder.java

```diff
@@ -35,8 +38,9 @@
      *
      * @param bindingType the binding type
      */
-    public SAML2LogoutRequestBuilder(final String bindingType) {
-        this.bindingType = bindingType;
+    public SAML2LogoutRequestBuilder(final SAML2Configuration cfg) {
+        this.bindingType = cfg.getSpLogoutRequestBindingType();
+        this.useNameQualifier = cfg.isUseNameQualifier();
```

Comment:
Could you please elaborate on how this flag is used today?

Reply:
The flag is used on login to work around a SAML spec compliance bug that causes ADFS logins to fail:

https://github.com/pac4j/pac4j/blob/d1cd6b89eae6d17e529a336451630319072940c3/pac4j-saml/src/main/java/org/pac4j/saml/sso/impl/SAML2AuthnRequestBuilder.java#L164

This patch uses the same flag to fix the same SAML spec compliance bug that causes ADFS logouts to fail.

See the following thread: https://groups.google.com/forum/?hl=en#!topic/pac4j-users/A08HzzY0KBQ